var/spack/repos/builtin/packages/madis/package.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Madis(MakefilePackage):
+    """MADIS: The Meteorological Assimilation Data Ingest System (MADIS) is
+    dedicated toward making value-added data available from the National
+    Oceanic and Atmospheric Administration's (NOAA) Earth System Research
+    Laboratory (ESRL) Global Systems Division (GSD) (formerly the Forecast
```

Comment:
This affiliation is no longer correct. The Global Systems Laboratory (GSL) is now its own laboratory, directly under OAR, see https://gsl.noaa.gov/

Reply:
I'd also remove the `(formerly the ... (FSL))` part

Reply:
Okay, I copied and pasted from a different and more up to date website :)